doc/source/running.rst

```diff
@@ -6,78 +6,45 @@ Backing Up and Restoring Data
 .. contents::
    :local:
 
-.. _pbm.running.backup-coordinator:
-
-Running the Backup Coordinator
+Running |pbm|
 ================================================================================
 
-The |bc| can be executed on any server since it does not need a connection to a
-|mongod| instance. One |bc| is needed per deployment.
+The |pbm.app| program can be run on any server since it does not need a connection to a
```

Comment:
This was true for pbm-coordinator, but we do not have pbm-coordinator any more. I would just remove this.

Reply:
Agreed.